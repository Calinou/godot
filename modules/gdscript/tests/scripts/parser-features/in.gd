func test():
	assert("dot" in "Godot")
	assert(not "i" in "team")

	assert(true in [true, false])
	assert(not null in [true, false])
	assert(null in [null])

	assert(26 in [8, 26, 64, 100])
	assert(not Vector2i(10, 20) in [Vector2i(20, 10)])

	assert("apple" in { "apple": "fruit" })
	assert("apple" in { "apple": null })
	assert(not "apple" in { "fruit": "apple" })
