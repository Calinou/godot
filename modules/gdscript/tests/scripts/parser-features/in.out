GDTEST_OK
>> WARNING
>> Line: 2
>> ASSERT_ALWAYS_TRUE
>> Assert statement is redundant because the expression is always true.
>> WARNING
>> Line: 3
>> ASSERT_ALWAYS_TRUE
>> Assert statement is redundant because the expression is always true.
